src/test/kotlin/uk/gov/dluhc/printapi/service/aed/SearchDataSanitizationTest.kt

```diff
@@ -0,0 +1,85 @@
+package uk.gov.dluhc.printapi.service.aed
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+
+internal class SearchDataSanitizationTest {
+
+    @CsvSource(
+        value = [
+            "doe,DOE",
```

Comment:
Minor - do we need all these tests, and do we need so many examples for each "type" of sanitization?
Would 1 parameterized test like this do the same thing?
```
@CsvSource(
    value = [
        "doe,DOE",
        "  doe  ,DOE",
        "o'leary,OLEARY",
        "Llewelyn-Bowen,LLEWELYN BOWEN",
        "Harper  Adams,HARPER ADAMS"
    ]
)
```

Reply:
It's a copy paste from vca, so didn't really review these tests

Reply:
I think it's just testing the various aspects of sanitization but can merge them if you want